api/k8s/application/k8sApplicationRestHandler.go

```diff
@@ -89,9 +89,10 @@ type K8sApplicationRestHandlerImpl struct {
 	helmAppService         client.HelmAppService
 	userService            user.UserService
 	k8sCommonService       k8s.K8sCommonService
+	terminalEnvVariables   *util.TerminalEnvVariables
 }
 
-func NewK8sApplicationRestHandlerImpl(logger *zap.SugaredLogger, k8sApplicationService application2.K8sApplicationService, pump connector.Pump, terminalSessionHandler terminal.TerminalSessionHandler, enforcer casbin.Enforcer, enforcerUtilHelm rbac.EnforcerUtilHelm, enforcerUtil rbac.EnforcerUtil, helmAppService client.HelmAppService, userService user.UserService, k8sCommonService k8s.K8sCommonService, validator *validator.Validate) *K8sApplicationRestHandlerImpl {
+func NewK8sApplicationRestHandlerImpl(logger *zap.SugaredLogger, k8sApplicationService application2.K8sApplicationService, pump connector.Pump, terminalSessionHandler terminal.TerminalSessionHandler, enforcer casbin.Enforcer, enforcerUtilHelm rbac.EnforcerUtilHelm, enforcerUtil rbac.EnforcerUtil, helmAppService client.HelmAppService, userService user.UserService, k8sCommonService k8s.K8sCommonService, validator *validator.Validate, terminalEnvVariables *util.TerminalEnvVariables) *K8sApplicationRestHandlerImpl {
```

Comment:
i dont think this should work since we do not have a provider for terminalEnvVariables, instead will need to import EnvironmentVariables here and set terminal in our impl

Reply:
make should also break with this

Reply:
done